biorxiv/crawl_download_pdf.py

```diff
@@ -55,7 +60,7 @@ def crawl_and_download_pdf(category, output_folder="downloaded_pdf_files", max_p
     with requests.Session() as session:
         while current_page < max_pages:
```

Comment:
should the max page be included (because current_page is set to 1)? i.e `while current_page <= max_pages`

Reply:
@puja-trivedi the max pages grows based on the number of publications submitted to biorxiv and is the number of paginated pages